### PR TITLE
Add links for k8s trainings

### PR DESCRIPTION
**Pull request by afgane**

### CODE REVIEW

**Review by nuwang — APPROVED** [review]

File: group_vars/all.yaml [hunk 1]
```diff
@@ -41,6 +41,10 @@ gxy_io_rewrites:
       - /gatdiscussion/
       - /gat-questions
       - /gatQandA
+  - src: /k8s/deploy
+    dest: https://training.galaxyproject.org/training-material/topics/admin/tutorials/k8s-deploying-galaxy/tutorial.html
+  - src: /k8s/manage
+    dest: https://training.galaxyproject.org/training-material/topics/admin/tutorials/k8s-managing-galaxy/tutorial.html
```
Comment (hexylena) [inline group_vars/all.yaml:47]: (This is just as FYI, I definitely understand wanting these more memorable short URLs)

These additionally have existing PURLs (https://gxy.io/GTN:T00013, https://gxy.io/GTN:T00014) as well as automatic GTN short URLs based on topic/tutorial IDs (https://gxy.io/GTN:admin/k8s-deploying-galaxy, https://gxy.io/GTN:admin/k8s-managing-galaxy) that we don't do a very good job of exposing in an author-useful manner. If you have looked for such a feature anywhere on the GTN and not found it, let me know and we can add this information in a useful place there.

Comment (nuwang) [reply group_vars/all.yaml:47]: Thanks for the heads up! I think the GTN URL would have worked great. We (or at least I) never thought the feature existed. Very cool! This was used for a paper submission, so if an opportunity presents itself to fix this url, we'll update it to use the GTN link.

Comment (hexylena) [reply group_vars/all.yaml:47]: Sounds good!  We'll see if we can mention it in more places that are author-visible, I've added an FAQ for it as well.

(but for a paper, again _totally understand,_ /k8s/deploy and /k8s/manage are very user-friendly to type!)

Comment (afgane) [reply group_vars/all.yaml:47]: Now that you mentioned it, I recall you mentioning it during GCC but I forgot yesterday. I'm sure we'll have an opportunity to swap those URL before the final paper version. Thanks for pointing this out.

**Pull request merged**